src/components/Home.style.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex !important;
```

Comment:
Why do you need `!important` here?

`!important` should only be used in very rare cases. For example when you need to overwrite third-party CSS. But if you need to use it to overwrite your own styles it's a code smell.

Reply:
I'm not entirely sure what caused this conflict. I agree `!important` should be very rarely used. I checked again, and it isn't needed! It was a ghost from when I used Sass, and for some reason I needed it in the Sass component; apparently not anymore! I'll push that now.